pkg/xds/server/callbacks/dataplane_status_tracker.go

```diff
@@ -136,6 +148,20 @@ func (c *dataplaneStatusTracker) OnStreamRequest(streamID int64, req util_xds.Di
 		// Infer the Dataplane ID.
 		if proxyId, err := core_xds.ParseProxyIdFromString(req.NodeId()); err == nil {
 			state.dataplaneId = proxyId.ToResourceKey()
+
+			log := statusTrackerLog.WithValues(
+				"proxyName", state.dataplaneId.Name,
+				"mesh", state.dataplaneId.Mesh,
+				"streamID", streamID,
+				"type", md.GetProxyType(),
+				"dpVersion", md.GetVersion().GetKumaDp().GetVersion(),
+				"subscriptionID", state.subscription.Id,
+			)
+			if statusTrackerLog.V(1).Enabled() {
+				log = log.WithValues("node", req.Node())
+			}
+			log.Info("proxy connected")
```

Comment:
maybe instead of 
```
proxy connected	{"proxyName": "redis"...
```
it can be
```
connected {"type": "dataplane", "name": "redis"...
```
so we could easily see if this is dataplane or zoneingress or zoneegress

Reply:
we already log the type `"type", md.GetProxyType(),` in 156 line. I'd rather keep "proxy connected" instead of "connected". 

It's proxyName instead of name for consistency, see https://github.com/kumahq/kuma/pull/4436#discussion_r891458867